README.md

```diff
@@ -24,6 +24,7 @@ Recently updated rates will be shown in **bold**, and the embed title will be au
 - [Prerequisites](https://github.com/FM-17/poglink/blob/main/docs/prerequisites.md)
 - [Running natively in Python](https://github.com/FM-17/poglink/blob/main/docs/native-installation.md)
 - [Running within Docker container](https://github.com/FM-17/poglink/blob/main/docs/docker-installation.md)
+- [Installing as a `systemd` Service](https://github.com/FM-17/poglink/blob/main/docs/systemd-installation.md)
```

Comment:
Apparently I rolled back a bit too far and some of your changes were somehow included in this branch.

Reply:
You'd probably be well off to merge `main` into `fm-server-generalization` (or rebase, but that can be messier), because you're currently "7 commits ahead, 6 commits behind" according to the "branches" page on this repo.

Reply:
Oof yeah nvm, the diff is all weird regardless haha. Just tried the merge myself and it was still just as funky. Don't know what's going on here with the PR diff